atlp-rwanda/dynamites-ecomm-be
# Fixed user type

## Description

**Enhancement:** This pull request addresses the issue where the Get User Method was not returnig  the  user type 

## Changes Made

1. **Backend Changes:**
   - Updated Get user Controller
   - Handled error on Momo Payment when the  total amount is  Zero.



## How to Test

1. Make Request to get User.
2. Check  if you sent zero Total amount on Momo Pay is returning a error message.

## Related Issues
- #142
